packages/meteor/debug.js

```diff
@@ -39,6 +39,11 @@ Meteor._debug = function (/* arguments */) {
 
         if (allArgumentsOfTypeString)
           console.log.apply(console, [Array.prototype.join.call(arguments, " ")]);
+        // We want to print out the stack trace when dynamics_nodejs.js throws
+        // an exception. Need to know what case this is and then make it into
+        // array so that we can pass it to console.log.apply.
+        else if (arguments.length === 2)
+          console.log.apply(console, [arguments[1]]);
```

Comment:
I've stepped through the function calls using node-inspector and what I got was that it is thrown here[https://github.com/meteor/meteor/blob/devel/packages/meteor/dynamics_nodejs.js#L113] and then results in going to `Meteor._debug` at which point the arguments no longer contain the stack trace. I didn't see any other intermediary functions in the middle, and from our earlier conversation, that seemed to be because `onException()` changes with it's use around the codebase.